_topic_map.yml

```diff
@@ -195,16 +195,21 @@ Topics:
 - Name: Managing user access
   Dir: manage-user-access
   Topics:
-  - Name: Configuring Okta Identity Cloud as a SAML 2.0 identity provider
-    File: configure-okta-identity-cloud
-  - Name: Configuring Google Workspace as an OIDC identity provider
-    File: configure-google-workspace-identity
-  - Name: Configuring OpenShift Container Platform OAuth server as an identity provider
-    File: configure-ocp-oauth
   - Name: Managing RBAC in Red Hat Advanced Cluster Security for Kubernetes 3.63 and newer
     File: manage-role-based-access-control-3630
   - Name: Enabling PKI authentication
     File: enable-pki-authentication
+  - Name: Authentication provider structure
+    File: authentication-provider-structure
+  - Name: Configure identity provider
```

Comment:
**nitpick:** Configuring identity providers

- We use a gerund for all other topics
- More than one providers